x-pack/plugins/index_management/server/lib/fetch_indices.ts

```diff
@@ -19,6 +19,8 @@ async function fetchIndicesCall(
   const { body: indices } = await client.asCurrentUser.indices.get({
     index: indexNamesString,
     expand_wildcards: ['hidden', 'all'],
+    // only get specified properties in the response
```

Comment:
@yuliacech Could you add some more details for this comment to make a note that we want to avoid bloating this response, and that if we include mappings we risk exceeding the 500MB (or whatever it is) limit on Node payloads? The goal we want to achieve is to ensure someone making changes to this code doesn't accidentally reintroduce this bug.